api/report.go

```diff
@@ -2,17 +2,32 @@ package api
 
 // Report contains the fields of a Preflight report
 type Report struct {
-	// Unique ID of the report
+	// Unique ID of the report.
 	ID string `json:"id"`
-	// Timestamp indicates when the report was generated
+	// PreflightVersion indicates the version of preflight this report was generated with.
+	PreflightVersion string `json:"preflight-version"`
+	// Timestamp indicates when the report was generated.
 	Timestamp Time `json:"timestamp"`
-	// Cluster indicates which was the target of the report
+	// Cluster indicates which was the target of the report.
 	Cluster string `json:"cluster"`
-	// Package indicates which package was used for the report
-	Package     string          `json:"package"`
```

Comment:
Actually, in order to keep better backwards compatibility, let's not repurpose the `Package` property.

Let's keep `Package string` and create a new `PackageInformation Package`.

WDYT @wwwil ?

Reply:
Yeah I think it's better to avoid breaking change. Should the `Package` struct also be renamed `PackageInformation`?

Reply:
Yes, that's what I have done.